app/src/main/kotlin/com/fernandocejas/sample/features/movies/MovieView.kt

```diff
@@ -19,7 +19,7 @@ import android.os.Parcel
 import com.fernandocejas.sample.core.platform.KParcelable
 import com.fernandocejas.sample.core.platform.parcelableCreator
 
-data class MovieView(val id: Int, val poster: String) : KParcelable {
+data class MovieView(val id: Int, val poster: String?) : KParcelable {
```

Comment:
why is this nullable now?

Reply:
hi @Zhuinden it was because I was having a compiler error when passing 

>  parcel.readString() (which is nullable) 

to the constructor:
`constructor(parcel: Parcel) : this(parcel.readInt(), parcel.readString())`

Let me know your thoughts. Thanks!

Reply:
hmm I guess if he uses `KParcelable` and `parcelableCreator` instead of `@Parcelize` then it is unavoidable